scripts/jenkins/cloud/ansible/group_vars/all/all.yml

```diff
@@ -75,6 +76,7 @@ update_after_deploy: False
 maintenance_updates_path:
   Cloud: "SUSE_Updates_{{ cloud_repo_path|default('OpenStack-Cloud') }}_{{ sles_cloud_version[ansible_distribution_version]|default('9') }}_x86_64"
   SLES: "SUSE_Updates_SLE-SERVER_{{ ansible_distribution_major_version }}-SP{{ ansible_distribution_release }}_x86_64"
+  SLES-LTSS: "SUSE_Updates_SLE-SERVER_{{ ansible_distribution_major_version }}-SP{{ ansible_distribution_release }}-LTSS_x86_64"
```

Comment:
We don't need to look for LTSS repositories because LTSS SLE package updates are also being released into the Cloud update channel.

Reply:
removed